linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/SJSGen.scala

```diff
@@ -113,10 +113,10 @@ private[emitter] final class SJSGen(
   }
 
   def usesUnderlyingTypedArray(elemTypeRef: NonArrayTypeRef): Boolean =
-    getArrayUnderlyingTypedArrayClassRef(elemTypeRef)(Position.NoPosition).nonEmpty
+    getArrayUnderlyingTypedArrayClassRef(elemTypeRef)(GlobalRefTracking.Dangerous, Position.NoPosition).nonEmpty
```

Comment:
Why is the tracking not implicitly provided here? Probably worth a comment?

Reply:
Comment added.